README.md

```diff
@@ -28,28 +32,50 @@ This project is maintained by [@charlie-sanders](https://github.com/charlie-sand
 pip install pyprql
 ```
 
-### Try it out
 
-#### CSV file
+### Try out the Pandas integration
+
+```python
+import  pandas as pd 
+import pyprql.pandas
+df = (...)
+results_df = df.prql.query('from df | select [age,name,occupation] | filter age > 21')
+
+```
```

Comment:
Should we mention this uses duckdb under-the-hood? Or that's an implementation detail?

Reply:
We could use it to call out its [performance](https://duckdb.org/2021/05/14/sql-on-pandas.html) ?  Might be cool to do our own benchmarks with prql involved

Reply:
Yes good idea

I imagine the performance is basicially the duckdb performance, so we could could link to those if they're available?

Reply:
Well its kind of buried in https://duckdb.org/2021/05/14/sql-on-pandas.html .  Maybe I could do a blog post of making a benchmark for pandas native and pandas prql, with pretty graphs.  But be upfront about the performance being all duckdb with links to it.

Reply:
Yes! Particularly if it's also how the PRQL API is nice — then it can make a case for using this (rather than duckdb with sql). Could be great!